packages/ag-charts-enterprise/src/features/error-bar/errorBarNode.ts

```diff
@@ -29,17 +30,46 @@ export type ErrorBarCapFormatter = (params: AgErrorBarFormatterParams) => AgErro
 
 type CapDefaults = NonNullable<ErrorBarNodeDatum['capDefaults']>;
 
+function toPixelRange(range: InteractionRange): PixelSize {
+    return typeof range === 'string' ? 0 : range;
+}
+
+class HierarchialBBox {
```

Comment:
See my other comment, but I'm thinking these behaviours are not error-bar specific?

Reply:
We can safely make `HierarchialBBox` part of the ag-chart-community package as a utility class. Although I don't see much benefit to this. The `HierarchialBBox` class only supports a tree of depth 1, so it's not such a useful utility class. A more useful version of `HierarchialBBox` would support trees with multiple levels:
```diff
 class HierarchialBBox {
     private readonly union: BBox;
-    private readonly components: BBox[];
+    private readonly components: HierarchialBBox[];
     ...
 }
```

Reply:
What shall we do with this `HierarchialBBox` class? I can move it into the community package as a scene utility and have `ErrorBarNode` extend it implement hit testing. Please let me know what you think.